docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -53,6 +53,17 @@ Whenever possible, we've taken this opportunity to use the best data type for st
 
 ![Upgrade Path](./assets/airbyte_destinations_v2_upgrade_prompt.png)
 
+:::caution Pay attention to the following before upgrade
+
+* The upgrading process entails hydrating the v2 format raw table by querying the v1 raw table through a standard query, such as "INSERT INTO v2_raw_table SELECT * FROM v1_raw_table." 
+The duration of this process can vary significantly based on the data size and may encounter failures contingent on the Destination's capacity to execute the query. 
+In some instances, establishing a new connection could potentially expedite the process. 
```

Comment:
> In some instances, establishing a new connection could potentially expedite the process. 

I don't understand this line.  Perhaps you are sying:

```suggestion
In some cases, creating a new Airbyte connection, rather than migrating your existing connection, may be faster. Note that in these cases, all data will be re-imported. 
```